packages/app/src/components/time-series-chart/logic/legend.tsx

```diff
@@ -17,16 +19,30 @@ export function useLegendItems<T extends TimestampedValue>(
       shapeComponent: <SeriesIcon config={x} />,
     }));
 
+    /**
+     * Maximum number of legend items
+     *
+     * Used to determine if there are enough items to render a legend
+     */
+    let maxNumItems: number = items.length;
```

Comment:
nit; type will be inferred from `items.length`
```suggestion
    let maxNumItems = items.length;
```